app/models/game.rb

```diff
@@ -4,7 +4,7 @@ class Game < ActiveRecord::Base
   has_many :team_games
   has_many :teams, :through => :team_games
 
-  attr_accessible :white_score, :blue_score, :played_positions_attributes
+  # attr_accessible :white_score, :blue_score, :played_positions_attributes
```

Comment:
Leaving these commented out here so I remember to go back in and set up strong params .